test/services/book_search_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+require_relative '../lib/fake_api.rb'
+
+class BookSearchTest < ActiveSupport::TestCase
+  test "should return a hash if success" do
+    service = BookSearch.new(111, 'fake_api').call
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.